charts/app-config/values-integration.yaml

```diff
@@ -2253,9 +2253,11 @@ govukApplications:
         enabled: false
       workerEnabled: true
       workers:
-        - command: ['bin/rails', 'message_queue:consume_published_documents']
-          name: published-documents-consumer
+        - command: ['bin/rake', 'publish_event_pipeline:process_messages']
```

Comment:
Oh actually what am I even saying — it was already using Rake anyway. Ignore me 😅

Reply:
Ah yes 😅  We _have_ changed it to use `rake` as an invocation just in case we want to further decouple it from the Rails side of things in the future, but yes both of these will just run Rake under the hood anyway!